### DISCUSSION TITLE
No File Management

### DISCUSSION THREAD

**Discussion by jbreske** [forkineye/ESPixelStick#587] [Q&A]
Update: Not sure why my install was different than others I had observed.  Took a chance and 1) reformatted my card from my laptop.  It was already purchased and verified as FAT32.  2) Transferred an xlights fseq file from my laptop to the card.

I flashed my fourth AE+ board for install, installed the above card and powered on.  File Management was present.

I have no "file management" option with my Dig Quad AE+ board.  I pulled the latest developmental release from the google drive.  The flash tool hardware drop down has an AE+ choice.  I have four new AE+ boards. Four new san disk 32gb SD cards and 2 older 32gb SD cards.  All FAT 32.  The SD pin selections appear correct for the AE+ board.  I can drive espixelstick effects and xlights.  I can upload controller settings from xlights.  I can access the SD cards with my laptop.

Is there another qualifier besides just an installed SD card to get "file management"?  I've seen three videos that demonstrate the ability to use espixelstick with the AE+ board for FPP Remote.  Seems pretty straight forward.  I never get the "file management" option once flashed for AE+ and an SD card installed.

Am I missing something else?  Thanks in advance for any insights.

**Comment by MartinMueller2003** [comment]
Having an SD card installed is all that is needed. How did you determine that the gpio pins were correct?

**Comment by jbreske** [comment]
Quindor has a table for the pins on his website. The numbers in espixelstick matched his table everytime i flashed. Everything thing I saw said just install the SD card but that wasn't working.

Sent from Yahoo Mail on Android 
 
  On Wed, Nov 23, 2022 at 12:59 PM, Martin ***@***.***> wrote:   


Having an SD card installed is all that is needed. How did you determine that the gpio pins were correct?

**Comment by MartinMueller2003** [comment]
Put the ESP into admin mode and go to the device config page. Verify the pins are correct on that page.

**Reply by Billybojangles01** [reply]
I did as you suggested but it won't allow me to change the 
SDcard - MISO pin to GPIO36 as required for the dig octa. Any ideas? Looks like the highest number it will accept is 24.

**Reply by MartinMueller2003** [reply]
I am pretty sure I have now fixed this in the UI. You can download the latest image at the same link

**Comment by mghaff007** [comment]
I have a similar issue with my esp-8266 micro sd card's. Anything over 4GB fat 32 my espixelstick version 4.0-beta4 doesn't see. But if i reduce the FS of the card to a 4GB fat32 partition I do get the file management tab. Do I have something set wrong? or does the esp8266 only see up to 4GB. Any responce is greatly appreciated. 

Thanks

**Reply by MartinMueller2003** [reply]
I have not seen this. I always use a 16gb card to test

**Reply by mghaff007** [reply]
Martin is that with a esp32? or esp-8266 or both? What model esp's are you using? I only ask because I have esp-8266-07S 4MB  and esp-8266-12E 4MB flash that we are using and neither will work with my formatted 16GB FAT32 cards without partitioning a 4GB FAT32 for 4.0 beta 4 to see the card. Still waiting for my esp-32's that I ordered to test my cards on. I'm pretty sure it shouldn't matter, but what brand and model cards are you using? 

Thanks in advance.

**Comment by MartinMueller2003** [comment]
For ESP8266 I have the LoLin D1 R1 Pro and the D1 mini. 
ESP32 I have LoLin D32 Pro, Twilight Lord, QuinLed D1 Mini, QuinLed Quad, DevKit C V4

The code for the two implementations is common. The difference is in the IDF. Each Esppressif line has its own IDF and I do not know all of the differences. Also, there have been bug fixes made. You can always get the latest build here: 
https://drive.google.com/file/d/1DG3wz8lbYNc371OIbeOqkzMaY5Zd2SKA/view?usp=share_link

**Comment by MartinMueller2003** [comment]
Updated Link
https://drive.google.com/drive/folders/1bGrkXSMoGjWtX5iWspPFRogM2G6c1VXu?usp=sharing
